ironic/drivers/modules/drac/inspect.py

```diff
@@ -44,7 +93,261 @@ class DracRedfishInspect(redfish_inspect.RedfishInspect):
     specific incompatibilities and introduction of vendor value added
     should be implemented by this class.
     """
-    pass
+
+    def inspect_hardware(self, task):
+        """Inspect hardware to get the hardware properties.
+
+        Inspects hardware to get the essential properties.
+        It fails if any of the essential properties
+        are not received from the node.
+
+        :param task: a TaskManager instance.
+        :raises: HardwareInspectionFailure if essential properties
+                 could not be retrieved successfully.
+        :returns: The resulting state of inspection.
+
+        """
+        node = task.node
+        system = redfish_utils.get_system(node)
```

Comment:
It looks like a lot of the code below was copied from RedfishInspect.inspect_hardware().  Instead of copying all that code, could we just call: super(DracRedfishInspect, self).inspect_hardware()?  We could let it create the Ports, and then set pxe_enabled on the ports after.

Reply:
Good idea! Done.